config/config-package.py

```diff
@@ -169,6 +170,25 @@ def _get_options_for(self, section, names):
             options[name] = self.cfg_option(section, name, '')
         return options
 
+    def _test_cfg(self):
+        """Setup testing configuration."""
+        options = self._get_options_for(
+            'test',
```

Comment:
I decided to add a new key because this could affect other files as well. 
Usually with pytest we could set some default settings on project.toml or change some flake/black configuration.